www/src/scss/openedx-theme.scss

```diff
@@ -1,4 +1,5 @@
 @import "~paragon-style/scss/core/core";
 @import "~paragon-style/css/themes/light/variables";
 @import "~paragon-style/css/themes/light/utility-classes";
+@import "~paragon-style/css/themes/light/abstraction-variables";
```

Comment:
Should this be rearranged as per https://github.com/openedx/paragon/pull/3222#discussion_r1770577421 ?

Reply:
We now use the name `abstraction-variables` for core variables and theme-related variables.

Reply:
This comment was about the import order, not the naming

Reply:
GitHub didn't note this as outdated, looks like it's in the proper order now! Thanks!